Cython/Includes/cpython/unicode.pxd

```diff
@@ -111,6 +111,18 @@ cdef extern from *:
     # UTF-8 encoded bytes.  The size is determined with strlen().
     unicode PyUnicode_FromString(const char *u)
 
+    unicode PyUnicode_New(Py_ssize_t size, Py_UCS4 maxchar)
+    unicode PyUnicode_FromKindAndData(int kind, const void *buffer, Py_ssize_t size)
+    unicode PyUnicode_FromFormat(const char *format, ...)
+    Py_ssize_t PyUnicode_GetLength(object unicode) except -1
```

Comment:
Is it a problem for Cython to have a variable name here similar to a type? Other definitions in this file already use it as variable name as well.

Reply:
I don't think this is an issue

Reply:
The parser needs to be rather smart regarding types since Cython allows you to leave out either the type (because of Python) or the argument name (because of C). Therefore it can handle this case just fine.